test/controllers/beneficial.owner.type.controller.spec.ts

```diff
@@ -202,13 +209,54 @@ describe("BENEFICIAL OWNER TYPE controller", () => {
       expect(resp.header.location).toEqual(config.MANAGING_OFFICER_CORPORATE_URL);
     });
 
-    test("renders the current page with error message", async () => {
+    test("renders the current page with beneficial owner error message", async () => {
+      mockGetApplicationData.mockReturnValueOnce({ beneficial_owners_statement: BENEFICIAL_OWNER_STATEMENT_OBJECT_BO_MOCK });
       const resp = await request(app).post(config.BENEFICIAL_OWNER_TYPE_URL);
 
       expect(resp.status).toEqual(200);
-      expect(resp.text).toContain(BENEFICIAL_OWNER_TYPE_PAGE_HEADING);
+      expect(resp.text).toContain(BENEFICIAL_OWNER_TYPE_PAGE_HEADING_ALL_IDENTIFIED_ALL_DETAILS);
```

Comment:
Could you please remove the following unit tests, they are not testing the presence of error message but the legend texts [here](https://github.com/companieshouse/overseas-entities-web/blob/c3526f1060def8adb484413a8ccb2baf7ae13903/views/beneficial-owner-type.html#L36)
- renders the current page with beneficial owner error message
- renders the current page with managing officer error message

Reply:
Looks like thee are a few other changes to make from deleting these tests and adding the ones above. Reverted the mock values and updated the some identified test.